scio-schemas/src/main/avro/schema.avsc

```diff
@@ -63,6 +64,7 @@
         {"name": "float_field", "type": ["null", "float"], "default": null},
         {"name": "double_field", "type": ["null", "double"], "default": null},
         {"name": "boolean_field", "type": ["null", "boolean"], "default": null},
+        {"name": "bytes_field", "type": ["null", "bytes"], "default": null},
         {"name": "string_field", "type": ["null", "string"], "default": null},
         {"name": "array_field", "type": {"type": "array", "items": "string"}, "default": null}
```

Comment:
We are also missing a `map_field` and a `fixed_field`. Would you also like to add those ?
I'm seeing for array that the default value is not correct. this should be set to `[]`. IIRC this won't be accepted anymore when updating avro version